examples/js/loaders/GLTFLoader.js

```diff
@@ -1370,11 +1550,13 @@ THREE.GLTFLoader = ( function () {
 								? THREE.QuaternionKeyframeTrack
 								: THREE.VectorKeyframeTrack;
 
+							var targetName = node.name ? node.name : node.uuid;
```

Comment:
Yes, I misread. My concern with UUID (mentioned in #10366) is that cloned meshes will be incompatible with the original AnimationClip objects, and there isn't (and really shouldn't be) a mechanism for cloning an AnimationClip. The same concern applies to a direct mapping, because you still can't clone the object easily.

But let's keep what you have here, and deal with #10366 later as you suggest. I think we can find a good solution later.